examples/hf_trainer_api/hf_image_classification/image_classification.py

```diff
@@ -312,7 +312,10 @@ def main(det_callback, tb_callback, model_args, data_args, training_args):
         id2label[str(i)] = label
 
     # Load the accuracy metric from the datasets package
-    metric = datasets.load_metric("accuracy", trust_remote_code=True,)
+    metric = datasets.load_metric(
```

Comment:
that's an odd random reformat. maybe removing the last comma would've been okay as well?